fission/src/systems/simulation/wpilib_brain/SimInput.ts

```diff
@@ -0,0 +1,70 @@
+import World from "@/systems/World"
+import EncoderStimulus from "../stimulus/EncoderStimulus"
+import { SimCANEncoder, SimGyro } from "./WPILibBrain"
+import Mechanism from "@/systems/physics/Mechanism"
+import Jolt from "@barclah/jolt-physics"
+import JOLT from "@/util/loading/JoltSyncLoader"
+
+export interface SimInput {
+    Update: (deltaT: number) => void
+}
+
+export class SimEncoderInput implements SimInput {
+    private _device: string
+    private _stimulus: EncoderStimulus
+
+    constructor(device: string, stimulus: EncoderStimulus) {
+        this._device = device
+        this._stimulus = stimulus
+    }
+
+    public Update(_deltaT: number) {
+        SimCANEncoder.SetPosition(`${this._device}`, this._stimulus.positionValue)
+        SimCANEncoder.SetVelocity(`${this._device}`, this._stimulus.velocityValue)
```

Comment:
Is there a reason for using template literals here when `_device` is already a string?

Reply:
Nope =)
It was incase some other formatting needed to be done later at the time.